bugToJira.py

```diff
@@ -0,0 +1,258 @@
+zendesk_instance = get_value("Zendesk instance: https://[yourZendeskInstance].zendesk.com: ",
+                             is_mandatory=True)
+
+# Load bugs from bugzilla
+print("\nLoading bugs...")
+cursor_bugs = mysql_conn.cursor()
+cursor_bugs.execute("""
+    SELECT
+    b.bug_id,
+    b.bug_severity,
+    b.bug_status,
+    b.creation_ts,
+    b.short_desc,
+    b.op_sys,
+    b.priority,
+    p.name product,
+    b.rep_platform,
+    b.version,
+    c.name component,
+    b.cf_bugtype,
+    b.cf_listusers,
+    b.cf_zendesk_ticket_id_text
+
+    FROM bugs b, products p, components c
+
+    WHERE
+    b.bug_status NOT IN ('RELEASED', 'CLOSED')
+    AND b.resolution IN ('')
+    AND b.product_id = p.id
+    AND b.component_id = c.id
+    AND p.name NOT IN ('Genymotion web site')
+
+    ORDER BY b.bug_id""")
+rows_bugs = cursor_bugs.fetchall()
+
+# Load android versions
+print("Loading android versions...")
+cursor_android_version = mysql_conn.cursor()
+cursor_android_version.execute("""
+    SELECT av.bug_id, av.value
+    FROM bug_cf_android_version av""")
+rows_android_version = cursor_android_version.fetchall()
+
+# Load attachments
+print("Loading attachments...")
+cursor_attachments = mysql_conn.cursor()
+cursor_attachments.execute("""
+    SELECT a.bug_id, a.filename, a.description, d.thedata
+    FROM bugs b, products p, attachments a, attach_data d
+    WHERE b.bug_id = a.bug_id
+        AND a.attach_id = d.id
+        AND b.product_id = p.id
+        AND b.bug_status NOT IN ('RELEASED', 'CLOSED')
+        AND b.resolution IN ('')
+        AND p.name NOT IN ('Genymotion web site')
+    ORDER BY a.bug_id""")
+rows_attachments = cursor_attachments.fetchall()
+
+# Load comments
+print("Loading comments...")
+cursor_comments = mysql_conn.cursor()
+cursor_comments.execute("""
+    SELECT c.bug_id, w.login_name, c.bug_when, c.thetext
+    FROM bugs b, products p, longdescs c, profiles w
+    WHERE b.bug_id = c.bug_id
+        AND c.who = w.userid
+        AND b.product_id = p.id
+        AND b.bug_status NOT IN ('RELEASED', 'CLOSED')
+        AND b.resolution IN ('')
+        AND c.thetext NOT IN ('')
+        AND p.name NOT IN ('Genymotion web site')
+    ORDER BY c.comment_id""")
+rows_comments = cursor_comments.fetchall()
+
+print("\nDisconnecting from your MySQL database...")
+mysql_conn.close()
+print("Disconnected.\n")
+
+# We will store a dict with bugzilla {bug id: jira issue} to, then, add
+# android versions, comments and attachments
+bug_id_jira_issue_dict = {}
+
+# Create JIRA issues from bugs (without comments, attachments and Android versions)
+print("Creating JIRA issue for each bug...\n")
+for row_bug in rows_bugs:
+    print("Loading bug {0} : {1}".format(row_bug[0], row_bug[4]))
+    issue_summary = '[BZ' + str(row_bug[0]) + '] ' + row_bug[4]
+    issue_description = '[CREATED]: ' + str(row_bug[3]) + '\n' + \
+                        '[Operating system]: ' + row_bug[5] + '\n' + \
```

Comment:
There's so many way to go to a new line in Python, you choose the worst one :p 

If think you should use 
```
'[CREATED]: ' + str(row_bug[3]) + '\n'
                       '[Operating system]: ' + row_bug[5] + '\n' + \
```

or even better: 

```
"""[CREATED]: {0}
    [Operating system]: {1}
""".format()
```